src/kafka-file/languageservice/services/codeLensProvider.ts

```diff
@@ -134,6 +135,12 @@ export class KafkaFileCodeLenses {
                     valueFormatSettings = this.getSerializationSettings(callee);
                     break;
                 }
+                case 'headers': {
+                    if (property.propertyValue) {
+                        this.parseHeaders(property.propertyValue, headers);

```

Comment:
Update parseHeader to write:

`headers = this.parseHeaders(property.propertyValue);`

Reply:
fixed